### PR TITLE
only add those intruments deployed

### PR DESCRIPTION
**Pull request by birdage**

### CODE REVIEW

File: ooiservices/app/main/alertsalarms.py [hunk 1]
```diff
@@ -139,6 +139,7 @@ def get_alert_alarm_status():
                 print "Ref-Des not in asset name list: ERROR: appending", aa_item['reference_designator']            
 
                 new_asset = {'ref_des':aa_item['reference_designator'],
+                             'hasDeploymentEvent' : True,
```
Comment (birdage) [inline ooiservices/app/main/alertsalarms.py]: need to add a deployment evert for ones created

File: ooiservices/app/main/alertsalarms.py [hunk 2]
```diff
@@ -160,49 +161,50 @@ def get_alert_alarm_status():
...
-            entry["event_type"] = status_outline[d]["event_type"]
-        #healthly 
-        elif d in aa_def_list:
-            #used to identify health sensors
-            entry["event_type"] = 'inactive'
-        #no status, unknown
-        else:
-            #nothing to do here
-            pass
+        if 'hasDeploymentEvent' in asset and asset['hasDeploymentEvent']:
```
Comment (birdage) [inline ooiservices/app/main/alertsalarms.py]: only allowed deployed ones

**Comment by DanielJMaher** [comment]
@birdage Merge Conflicts

**Comment by DanielJMaher** [comment]
@birdage

**Comment by birdage** [comment]
@DanielJMaher seems git had an issue with the compare, its says there are differences but it is the same content...

**Comment by birdage** [comment]
@DanielJMaher @FBRTMaka @Bobfrat ready for review and merge

**Pull request merged**